packages/framework/src/Model/Product/Search/Export/ProductSearchExportRepository.php

```diff
@@ -71,4 +72,16 @@ protected function createQueryBuilder(int $domainId, string $locale): QueryBuild
 
         return $queryBuilder;
     }
+
+    /**
+     * @param int $domainId
+     * @param string $locale
+     * @return int
+     */
+    public function getProductTotalCountForDomainAndLocale(int $domainId, string $locale): int
+    {
+        $result = new QueryPaginator($this->createQueryBuilder($domainId, $locale));
+
+        return $result->getTotalCount();
```

Comment:
cool trick. I thought it is slow, but it seems to be fast (only total count is fetched from DB, not paginated results) :+1:

Reply:
Thanks :)